File: programmers/python/입문100/18.2_제곱수판별하기★/18-2.py
# ** 0.5 : 제곱근 구하기
# int() 로 소숫점 버리고
# ** 2 : 다시 제곱헀을때 n 이면 제곱수

def solution(n):
    if int(n ** 0.5) ** 2 == n:
        return 1
    else:
        return 2

File: programmers/java/기초124/18.2_문자열잘라서정렬하기/18_2.java
import java.util.*;

class Solution {
    public String[] solution(String myString) {
        // String[] answer = {};
        // return answer;
        String[] parts = myString.split("x");
        List<String> list = new ArrayList<>();

        // 빈 문자열 제거
        for (String s : parts) {
            if (!s.isEmpty()) {
                list.add(s);
            }
        }

        // 사전순 정렬
        Collections.sort(list);

        // 배열로 변환
        return list.toArray(new String[0]);
    }
}